### PR TITLE
refactor(step-generation): fix and check circular dependencies

### PR DESCRIPTION
**Pull request by IanLondon**
# Overview

PR serves as its own ticket.

Adds `step-generation` to the `make circular-dependencies-js` check, and fixes a circular dependency

# Changelog

# Review requests

- Code review
- CI passes

# Risk assessment

Low

### CODE REVIEW

File: step-generation/src/robotStateSelectors.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // @flow
 import assert from 'assert'
 // TODO: Ian 2019-04-18 move orderWells somewhere more general -- shared-data util?
-import { orderWells } from './utils'
+import { orderWells } from './utils/orderWells'
```
Comment (IanLondon) [inline step-generation/src/robotStateSelectors.js:4]: importing "internally" from the `index.js` caused circular dep issues unfortunately 😞

**Review by Kadee80 — APPROVED** [review]
🛫

File: step-generation/src/robotStateSelectors.js (hunk 1 above)
Comment (shlokamin) [reply step-generation/src/robotStateSelectors.js:4]: yikes, looks like we have some more code organization work to do :/

Comment (shlokamin) [reply step-generation/src/robotStateSelectors.js:4]: this is probably an indication that our module boundaries aren't quite strict enough

**Review by shlokamin — APPROVED** [review]
thank you @IanLondon!

**Pull request merged**